### ISSUE TITLE
Document why `inert` is an option on the select element

### ISSUE THREAD

**Issue by maggiewachs** [department-of-veterans-affairs/vets-design-system-documentation#5020]
# Bug Report

- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened
I understand that this issue has [been discussed](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1526), but haven't found where a solution was provided. The buggy version of this feature is still in production code, in the [select component documentation](https://design.va.gov/components/form/select#read-only) and in a [mock form](https://staging.va.gov/mock-form-patterns/mailing-address).

I found this issue when reviewing code for a VA project that is using the VA DS form components.

In the mock form linked above, when the "I live on a U.S. military base" checkbox is checked, the Country field's label and value are displayed but are completely inaccessible to screen reader users because the `inert` attribute does not make text read-only, but visible-only — the result is more like an image of text. It's intended for content that should be completely unavailable to everyone, even when visible. A good example is when a modal dialog appears, any visible page content outside of the dialog should be inert. 

I don't think there's a code solution, as much as a documentation solution. To prevent inaccessible content (and an accessibility violation), the documentation on `inert` should make clear that inert elements should not provide any useful content or feedback to anyone. It would also help to retire examples like the Country field in the mock form because `inert` should not be used in place of making something read-only.

## What I expected to happen
Text displayed on the page would be read by VoiceOver.

## Reproducing

**URL(s) where this bug occurs**
https://staging.va.gov/mock-form-patterns/mailing-address

**Steps to reproduce:**

1. Start up a screen reader (I tested with VoiceOver)
2. Open the mock mailing address form: https://staging.va.gov/mock-form-patterns/mailing-address
3. Check the checkbox labeled "I live on a U.S. military base outside of the United States."
4. Try to "read" the Country field.

## Urgency

How urgent is this request? Please select the appropriate option below and/or provide details

- [ ] This bug is blocking work currently in progress
- [ ] This bug is affecting work currently in progress but we have a workaround
- [ ] This bug is blocking work planned within the next few sprints
- [ ] This bug is not blocking any work
- [x] Other — displaying inert text as "read-only" without providing text alternative is an accessibility violation

**Comment by humancompanion-usds** [comment]
IIRC, the a11y community weighed in on our use of inert and that is where we landed. Making the country field inert was not an [accident](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1526#issuecomment-2963521334). The [accessibility considerations section of the Select guidance](https://design.va.gov/components/form/select#accessibility-considerations) need to cover why we did what we did here.

**Comment by maggiewachs** [comment]
> IIRC, the a11y community weighed in on our use of inert and that is where we landed. Making the country field inert was not an [accident](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1526#issuecomment-2963521334). The [accessibility considerations section of the Select guidance](https://design.va.gov/components/form/select#accessibility-considerations) need to cover why we did what we did here.

Hi @humancompanion-usds — Yes, I was aware of that ticket and discussion; I linked to it in my comment. 

It's perfectly OK to present read-only feedback to users in a variety of contexts. But that read-only feedback must be accessible to everyone. The current pattern provides read-only Country feedback to only one set of users (sighted people) and it's completely inaccessible to another (blind people). It really doesn't matter that an explanation exists elsewhere on the page in an additional info disclosure. If you present text, it needs to be available to all users, and not doing so fails [WCAG SC 1.3.1 Info and Relationships (Level A)](https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships.html). 

The [MDN documentation cautions against using inert ](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Global_attributes/inert) because inert elements "are hidden from assistive technologies as they are excluded from the accessibility tree". The inert attribute was not meant to be used as read-only regardless of the good intent when implementing it, so I hope you'll consider making a correction.
